### ISSUE TITLE
strange discontinuities in the mask fit area

### ISSUE THREAD

**Issue by tturpin** [alessandro-zomparelli/myfacemask#6]
Hi,
I'm having trouble with the control points generated at the border : there are about 6 places along the contour where the line is "twisted", with lots of control points very close to each other (and thus many lines that "converge" from the filter to those points). Any idea what could cause this ?

I can mitigate it by playing with the filter position (though the effect quite unpredictable), but I cannot eliminate it. I also tried smoothing the face more, with no success. And my attempts to manually edit the mesh by moving any of the points were a disaster... Since I cannot change my face, I'm out of solutions to try :-(

![image](https://user-images.githubusercontent.com/781298/79643830-29420180-81a5-11ea-9bfe-2f1d56065099.png)

**Comment by alessandro-zomparelli** [comment]
Hi @tturpin thank you for reporting this issue. Did you scale the face after the remesh?

**Comment by tturpin** [comment]
It's possible. I did several tries, the first time, I definitely rescaled after remesh, after I realized that the mask was larger than my whole body. In my last attempt, I believe that I did the scaling first, but maybe I changed it again after remesh. I will retry and check that. Also I have rescaled the filter part of the mask itself, because it seemed a bit small (but maybe I scaled my face too large, I did a rough estimate from the distance between the eyes). It did not seem to break anything, except that the other part of the assembly would obviously need the same scaling. Is this supposed to work ? It would be easier if I the photogrammetry software could measure the size and orient the face correctly... (I'm using 3DF Zephyr Free).

**Comment by tturpin** [comment]
Retried with scaling only before remesh, now it's all good. I also get the correct behavior for the offset, which was affected by scaling previously (but the thickness was not, interestingly).
Thanks for the solution !

**Comment by alessandro-zomparelli** [comment]
Great, I will lock the scale of the remeshed face, just in case :-)
It's normal that the offset of the projection is affected by the target object, while the thickness is affected by the the scale of the object itself. For that reason is better not scaling the filter. Now it should be already locked.

**Comment by tturpin** [comment]
Thanks a lot. Now I just have to remember to apply scaling before anything else... Maybe it would be useful to give the user a hint about scaling. For example, representing a mask or a face somewhere so that a 20x scaling error becomes obvious ?

**Comment by alessandro-zomparelli** [comment]
Yes, this is actually a nice idea, I will think about an easy why to give some clues to the user.
